rest/tests/test_learning_resource.py

```diff
@@ -451,3 +547,123 @@ def test_static_assets_delete(self):
+        self.create_learning_resource_export(repo_slug1, {"id": repo1_lrid1})
+        self.assertEqual(
+            [{"id": repo1_lrid1}],
+            self.get_learning_resource_exports(
+                repo_slug1
+            )['results'])
+
+        # make sure export ids are confined to their own repos
+        self.create_learning_resource_export(repo_slug2, {
+            'id': repo2_lrid1
+        })
+        # not affected by POST in previous line
+        self.assertEqual([{'id': repo1_lrid1}],
+                         self.get_learning_resource_exports(
+                             repo_slug1)['results'])
+        # contains the new item
+        self.assertEqual([{'id': repo2_lrid1}],
+                         self.get_learning_resource_exports(
+                             repo_slug2)['results'])
+
+        # make sure we can't add export ids that don't belong to the repo
+        self.create_learning_resource_export(repo_slug1, {
+            'id': repo2_lrid1
+        }, expected_status=HTTP_403_FORBIDDEN)
+
+        # make sure detail view will 404 if out of bounds
+        self.assertEqual(repo1_lrid1, self.get_learning_resource_export(
+            repo_slug1, repo1_lrid1)['id'])
+        self.get_learning_resource_export(
+            repo_slug1, repo2_lrid1,
+            expected_status=HTTP_404_NOT_FOUND)
+        self.get_learning_resource_export(
+            repo_slug2, repo1_lrid1,
+            expected_status=HTTP_404_NOT_FOUND)
+        self.assertEqual(repo2_lrid1, self.get_learning_resource_export(
+            repo_slug2, repo2_lrid1)['id'])
+
+        # repo1_lrid1 belongs to repo1, can't add it to repo2
+        self.create_learning_resource_export(repo_slug2, {
+            "id": repo1_lrid1
+        }, expected_status=HTTP_403_FORBIDDEN)
+
+        self.create_learning_resource_export(repo_slug2, {"id": repo2_lrid1})
+        self.create_learning_resource_export(repo_slug2, {"id": repo2_lrid2})
+
+        # finally, delete all of the things
+        self.assertEqual(
+            2, self.get_learning_resource_exports(repo_slug2)['count'])
+        self.delete_learning_resource_exports(repo_slug2)
+        self.assertEqual(
+            0, self.get_learning_resource_exports(repo_slug2)['count'])
+
+        # make sure deleting an empty list doesn't cause problems
+        self.delete_learning_resource_exports(repo_slug2)
+        self.assertEqual(
+            0, self.get_learning_resource_exports(repo_slug2)['count'])
+
+        # repo1 is unaffected
+        self.assertEqual(
+            1, self.get_learning_resource_exports(repo_slug1)['count'])
+        self.delete_learning_resource_export(repo_slug1, repo1_lrid1)
+        self.get_learning_resource_export(
+            repo_slug1, repo1_lrid1, expected_status=HTTP_404_NOT_FOUND)
+
+        self.logout()
+        self.login(self.user)
+
+        # After logout and login session was cleared
+        self.assertEqual(
```

Comment:
Does this test anything? It looks like it might if line 660 didn't delete the cart. Right now, it looks like this `assertEqual` will always be true, even if there's a bug.